ui/packages/consul-ui/app/abilities/operator.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import BaseAbility from './base';
+
+export default class OperatorAbility extends BaseAbility {
+  resource = 'operator';
+}
```

Comment:
Would love some eyes on this, from Melissa, we need the `operator` write permission, is this enough?

By adding this I'm able to do `this.abilities.can('write operators')`, and it seems to be checking the right permissino

Reply:
lgtm